components/admin/actions.tsx

```diff
@@ -267,6 +51,15 @@ export default function Actions({ data }: PageProps): JSX.Element {
         'aboveZero',
         'Bedroom need should be a number greater than 0',
         (value, testContext) => {
+          console.log(value, testContext);
+
+          if (
+            testContext.parent.status !== ApplicationStatus.ACTIVE &&
+            testContext.parent.status !== ApplicationStatus.ACTIVE_UNDER_APPEAL
+          ) {
+            return true;
```

Comment:
This validation condition doesn't seem to match the error message: "Bedroom need should be a number greater than 0"

Reply:
The idea here is to skip validation (return true) when the selected status doesn't show the bedroom need question. Maybe there's a better way but I'm not sure what it is.